pom.xml

```diff
@@ -577,7 +577,7 @@
       <dependency>
         <groupId>${project.groupId}</groupId>
         <artifactId>netty-tcnative</artifactId>
-        <version>1.1.32.Fork1</version>
+        <version>1.1.32.Fork2-SNAPSHOT</version>
```

Comment:
Sorry for the delay.. quite busy last week and this. Just wanted to let you know that I still have it on my to do list. It is just not as easy as I hoped because of alpn which needs openssl 1.0.2